conn.go

```diff
@@ -161,6 +161,7 @@ func (c *Conn) Read(p []byte) (n int, err error) {
 		return 0, c.getConnErr()
 	}
 	if len(p) < len(out) {
+		fmt.Println(errBufferTooSmall)
```

Comment:
Do we need this change? Mind putting it in another PR (just in case we need to revert debug)

Reply:
Good catch 😳